docs/src/examples/cassandra-cluster-shotover-sidecar.md

```diff
@@ -0,0 +1,118 @@
+# Cassandra Cluster
+  
+The following guide shows you how to configure Shotover with support for proxying to a Cassandra Cluster.
+
+## Overview
+
+In this example, we will be connecting to a Cassandra cluster that has the following topology:
+
+* `172.16.1.2:9042`
+* `172.16.1.3:9042`
+* `172.16.1.4:9042`
+
+### Rewriting the peer ports
+
+Shotover will be deployed as a sidecar to each node in the Cassandra cluster, listening on `9043`. Use the following [docker-compose.yml](https://raw.githubusercontent.com/conorbros/shotover-proxy/cassandra-docs/shotover-proxy/examples/cassandra-rewrite-peers/docker-compose.yml) to run the Cassandra cluster and Shotover sidecars. In this example we want to ensure that all our traffic to Cassandra goes through Shotover.
+
+```console
+curl -L https://raw.githubusercontent.com/conorbros/shotover-proxy/cassandra-docs/shotover-proxy/examples/cassandra-rewrite-peers/docker-compose.yml --output docker-compose.yml
```

Comment:
remember to change this before checking in

Reply:
Yep, I'll leave these unresolved until I switch it before merging.